wormhole-connect/src/components/AlertBanner.tsx

```diff
@@ -35,18 +35,14 @@ function AlertBanner(props: Props) {
   const { classes } = useStyles();
   const [alertContent, setAlertContent] = useState(props.content);
```

Comment:
Would this state hook still be needed? You could use `props.content` directly now that you don't need to clear the value.